packages/system/objectstorage-controller/Makefile

```
@@ -1,11 +1,28 @@
 
 update:
 	rm -rf templates
-	mkdir templates
-	kubectl kustomize github.com/kubernetes-sigs/container-object-storage-interface-api > templates/crds.yaml
-	kubectl kustomize github.com/kubernetes-sigs/container-object-storage-interface-controller > templates/controller.yaml
-	sed -i 's/namespace: default/namespace: {{ .Release.Namespace }}/g' templates/controller.yaml
+	mkdir -p templates
+	kubectl kustomize github.com/kubernetes-sigs/container-object-storage-interface > templates/controller.yaml
+	sed -i 's/namespace: container-object-storage-system/namespace: {{ .Release.Namespace }}/g' templates/controller.yaml
+
+image: image-controller image-sidecar
```

checkmake (0.2.2)

[warning] Target "image" should be declared PHONY.

(phonydeclared)